internal/nexodus/ctlserver_unix.go

```diff
@@ -1,4 +1,4 @@
-//go:build linux
+//go:build unix
```

Comment:
I think this can be simplified to:

    //+build linux darwin 

And then you don't need to makefile tricks.

Reply:
Ha. I tried to figure that out and tried lots of incorrect ways — comma separated, colon, semicolon …